contrib/inventory_builder/inventory.py

```diff
@@ -37,7 +37,7 @@
 
 from collections import OrderedDict
 from ipaddress import ip_address
-from ruamel.yaml import YAML
+from ruamel_yaml import YAML
```

Comment:
With `ruamel.yaml` I was not able to call the python script on my fedora 32 host. But `ruamel_yaml` has worked for me.

Reply:
It _is_ ok on my system, but it seems to fail in your CI pipeline. Has anyone an idea on this?